crates/astria-sequencer/src/accounts/query.rs

```diff
@@ -177,6 +181,7 @@ async fn get_snapshot_and_height(storage: &Storage, height: Height) -> Result<(S
     Ok((snapshot, height))
 }
 
+#[instrument(skip_all)]
```

Comment:
```suggestion
#[instrument(skip_all, err)]
```

Reply:
I tried this, and it doesn't work because the function's error type doesn't implement the display trait. Is there another way around this that you'd like to do?